HTTP UA Client Hint headers (#11191)
mdn/content: files/en-us/web/http/headers/sec-ch-ua-platform-version/index.md

The **`Sec-CH-UA-Platform-Version`** [user agent client hint](/en-US/docs/Glossary/Client_hints#user-agent_client_hints) request header provides the version of the operating system on which the user agent is running. 

<table class="properties">
  <tbody>
    <tr>
      <th scope="row">Header type</th>
      <td>
        {{Glossary("Request header")}},
        {{Glossary("Client hints","Client hint")}}
      </td>
    </tr>
    <tr>
      <th scope="row">{{Glossary("Forbidden header name")}}</th>
      <td>yes</td>
    </tr>
  </tbody>
</table>

## Syntax

```http
Sec-CH-UA-Platform-Version: <version>
```

### Directives

- `<version>`
  - : The version string typically contains the operating system version in a string, consisting of dot-separated major, minor and patch version numbers.
     For example, `"11.0.0"`

     The version string on Linux is always empty.

## Examples

A server requests the `Sec-CH-UA-Platform-Version` header by including the {{HTTPHeader("Accept-CH")}} in a _response_ to any request from the client, using the name of the desired header as a token:

```http
HTTP/1.1 200 OK
Accept-CH: Sec-CH-UA-Platform-Version
```

> The client may then opt-in to providing the hint, and add the `Sec-CH-UA-Platform-Version` header to subsequent requests.

Author: ```suggestion
The client may choose to provide the hint, and add the `Sec-CH-UA-Platform-Version` header to subsequent requests.
```
